src/app/core/models/userrole.model.ts

```diff
@@ -1,6 +1,10 @@
 export class UserRole {
   static FEECLERK = 'bar-fee-clerk';
+  static FEECLERKDISPLAY = 'Fee Clerk';
```

Comment:
fees clerk role name and display name should be tied together. create a class UserRole with attributes as roleID and roleName
and initialise four objects of that.